xbmc/cores/RetroPlayer/rendering/RenderContext.cpp

```diff
@@ -84,6 +84,9 @@ static ESHADERMETHOD TranslateShaderMethod(GL_SHADER_METHOD method)
   {
   case GL_SHADER_METHOD::DEFAULT: return SM_DEFAULT;
   case GL_SHADER_METHOD::TEXTURE: return SM_TEXTURE;
+#if defined(HAS_GLES)
```

Comment:
A translate function shouldn't change behavior based on external params, it should always translate to the same value IMO

Reply:
`CRenderSystemGL::EnableShader()` should reject the GLES only value

Reply:
fine as is